misk-testing/src/main/kotlin/misk/service/ServiceTestingModule.kt

```diff
@@ -27,9 +27,6 @@ class ServiceTestingModule<T : Service> internal constructor(
         getProvider(wrappedServiceType.java),
         extraDependencies
     )).asSingleton()
-
-    // Bind the wrapped service directly so that application code can inject it
-    bind(wrappedServiceType.java)
```

Comment:
For my use case, the wrapped service is already bound via a custom provider, since it has qualified injections. This binding conflicts. I also don't think it's common that an application will need to inject the Service, and even if it does the app could just call `bind()` on the type itself.

Reply:
yea I probably agree that it's not common. you are probably going to break downstream repos though :)

Reply:
I checked, we seem to only have 1 usage of this method and it doesn't break.